toolkit/tools/imagecustomizer/container/imagecustomizer.Dockerfile

```diff
@@ -5,10 +5,12 @@ ARG BASE_IMAGE="mcr.microsoft.com/azurelinux/base/core:3.0"
 
 FROM ${BASE_IMAGE}
 RUN tdnf update -y && \
+   tdnf install -y azurelinux-repos-cloud-native && \
+   tdnf update -y && \
    tdnf install -y qemu-img rpm coreutils util-linux systemd openssl \
       sed createrepo_c squashfs-tools cdrkit parted e2fsprogs dosfstools \
       xfsprogs zstd veritysetup grub2 grub2-pc systemd-ukify binutils lsof \
-      python3 python3-pip jq && \
+      python3 python3-pip jq oras && \
```

Comment:
Future - I'm assuming when IC directly downloads the OCI we will use an SDK and this dependency will go away?